settings.gradle

```diff
@@ -29,3 +29,6 @@ include ':backend:jmap'
 include ':backend:demo'
 include ':plugins:openpgp-api-lib:openpgp-api'
 include ':cli:html-cleaner-cli'
+
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
```

Comment:
❌ This might be fine if we had a dedicated build engineer. But we don't. So I'd like us to use a conservative build config. That means no preview features (unless there's a very good reason).

Reply:
Maybe we can add a todo here?

Reply:
The problem isn't that we'll forget about removing this line. The issue is that we have better things to do than debugging failing builds. Not using experimental features reduces the risk of running into build issues.

Reply:
31ddac19c312266521636e1f4f7a8dc1f0cf01f7